messaging/src/main/java/org/axonframework/deadline/SimpleDeadlineManager.java

```diff
@@ -283,6 +301,18 @@ public Builder transactionManager(@Nonnull TransactionManager transactionManager
             return this;
         }
 
+        /**
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Note that the `builder` methods throughout also define the default values. Hence, I'd expect all the infra components where you have added this default field to have a change in the `builder`'s JavaDoc too.